ballista/core/proto/datafusion.proto

```diff
@@ -852,47 +988,54 @@ message ScalarValue {
   }
 }
 
-message Decimal128 {
+message Decimal128{
+  bytes value = 1;
+  int64 p = 2;
+  int64 s = 3;
+}
+
+message Decimal256{
   bytes value = 1;
   int64 p = 2;
   int64 s = 3;
 }
 
 // Serialized data type
-message ArrowType {
+message ArrowType{
   oneof arrow_type_enum {
-    EmptyMessage NONE = 1; // arrow::Type::NA
-    EmptyMessage BOOL = 2; // arrow::Type::BOOL
-    EmptyMessage UINT8 = 3; // arrow::Type::UINT8
-    EmptyMessage INT8 = 4; // arrow::Type::INT8
-    EmptyMessage UINT16 = 5; // represents arrow::Type fields in src/arrow/type.h
+    EmptyMessage NONE = 1;     // arrow::Type::NA
+    EmptyMessage BOOL = 2;     // arrow::Type::BOOL
+    EmptyMessage UINT8 = 3;    // arrow::Type::UINT8
+    EmptyMessage INT8 = 4;     // arrow::Type::INT8
+    EmptyMessage UINT16 = 5;   // represents arrow::Type fields in src/arrow/type.h
     EmptyMessage INT16 = 6;
     EmptyMessage UINT32 = 7;
     EmptyMessage INT32 = 8;
     EmptyMessage UINT64 = 9;
-    EmptyMessage INT64 = 10;
-    EmptyMessage FLOAT16 = 11;
-    EmptyMessage FLOAT32 = 12;
-    EmptyMessage FLOAT64 = 13;
-    EmptyMessage UTF8 = 14;
+    EmptyMessage INT64 = 10 ;
+    EmptyMessage FLOAT16 = 11 ;
+    EmptyMessage FLOAT32 = 12 ;
+    EmptyMessage FLOAT64 = 13 ;
+    EmptyMessage UTF8 = 14 ;
     EmptyMessage LARGE_UTF8 = 32;
-    EmptyMessage BINARY = 15;
-    int32 FIXED_SIZE_BINARY = 16;
+    EmptyMessage BINARY = 15 ;
+    int32 FIXED_SIZE_BINARY = 16 ;
     EmptyMessage LARGE_BINARY = 31;
-    EmptyMessage DATE32 = 17;
-    EmptyMessage DATE64 = 18;
+    EmptyMessage DATE32 = 17 ;
+    EmptyMessage DATE64 = 18 ;
     TimeUnit DURATION = 19;
-    Timestamp TIMESTAMP = 20;
-    TimeUnit TIME32 = 21;
-    TimeUnit TIME64 = 22;
-    IntervalUnit INTERVAL = 23;
-    Decimal DECIMAL = 24;
+    Timestamp TIMESTAMP = 20 ;
+    TimeUnit TIME32 = 21 ;
+    TimeUnit TIME64 = 22 ;
+    IntervalUnit INTERVAL = 23 ;
+    Decimal DECIMAL = 24 ;
```

Comment:
Looks like you applied a lot of formatting in this PR. These whitespaces seem off though.